### ISSUE TITLE
possible vulnerability in bl 4.1.0

### ISSUE THREAD

**Issue by vladmandic** [rvagg/bl#101]
`auditjs ossi` scan from <https://github.com/sonatype-nexus-community/auditjs> reports vulnerability:

```log
[22/217] - pkg:npm/bl@4.1.0 - 1 vulnerability found!
  Vulnerability Title:  CWE-200: Information Exposure
  ID:  01fa430e-ebce-4504-bcbd-1a58bd22b168
  Description:  An information exposure is the intentional or unintentional disclosure of information to an actor that is not explicitly authorized to have access to that information.
  CVSS Score:  4.3
  CVSS Vector:  CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N
  Reference:  https://ossindex.sonatype.org/vuln/01fa430e-ebce-4504-bcbd-1a58bd22b168?component-type=npm&component-name=bl&utm_source=auditjs&utm_medium=integration&utm_content=4.0.23
```

note: `npm audit` does not report any issues.

**Comment by mcollina** [comment]
Sonatype is wrong. By looking at https://www.npmjs.com/advisories/1555, it clearly states that 4.0.3 fixes this, so 4.1.0 does not have that vulnerability.

**Comment by vladmandic** [comment]
@mcollina thanks for the quick feedback
